src/lib/react/__tests__/useLocalMedia.unit.ts

```diff
@@ -0,0 +1,60 @@
+import { jest } from "@jest/globals";
+import { act, renderHook } from "@testing-library/react";
+import useLocalMedia from "../useLocalMedia";
+import LocalMedia from "../../LocalMedia";
+
+function oneof<T>(...array: T[]) {
+    return array[Math.floor(Math.random() * array.length)];
+}
+
+jest.mock("../../LocalMedia");
+const MockedLocalMedia = jest.mocked(LocalMedia);
+
+describe("useLocalMedia", () => {
+    beforeEach(() => {
+        MockedLocalMedia.mockClear();
+    });
+
+    it("should assume audio and video by default", () => {
+        renderHook(() => useLocalMedia());
+        expect(MockedLocalMedia.mock.calls[0]).toEqual([{ audio: true, video: true }]);
+    });
+
+    it("should use provided audio and video values", () => {
```

Comment:
I would say that we should aim for deterministic unit tests. Can we just test all the cases? These tests are super cheap.
````
    it.each([
        { audio: true, video: true },
        { audio: true, video: false },
        { audio: false, video: true },
        { audio: false, video: false },
    ])(`should use provided audio: $audio and video: $video values`, ({ audio, video }) => {
        renderHook(() => useLocalMedia({ audio, video }));
        expect(MockedLocalMedia.mock.calls[0]).toEqual([{ audio, video }]);
    });
````

Reply:
Yea, this was prob an overkill to have the random values. For some tests however, I have found this setup useful in the past.

Reply:
Cleaned up 👍